services/catalog/src/simcore_service_catalog/cli.py

```diff
@@ -24,3 +27,52 @@ def run():
         "$ uvicorn simcore_service_catalog.main:the_app",
         fg=typer.colors.BLUE,
     )
+
+
+@main.command()
+def echo_dotenv(ctx: typer.Context, *, minimal: bool = True) -> None:
+    """Generates and displays a valid environment variables file (also known as dot-envfile)
+
+    Usage:
+        $ simcore-service echo-dotenv > .env
+        $ cat .env
+        $ set -o allexport; source .env; set +o allexport
+    """
+    assert ctx  # nosec
+
+    # NOTE: we normally DO NOT USE `os.environ` to capture env vars but this is a special case
+    # The idea here is to have a command that can generate a **valid** `.env` file that can be used
+    # to initialized the app. For that reason we fill required fields of the `ApplicationSettings` with
+    # "fake" but valid values (e.g. generating a password or adding tags as `replace-with-api-key).
+    # Nonetheless, if the caller of this CLI has already some **valid** env vars in the environment we want to use them ...
+    # and that is why we use `os.environ`.
+
+    settings = ApplicationSettings.create_from_envs(
```

Comment:
I know that we often have example of valid data for the pydantic models in the `config` in the models. These are often tested, so that we know the data is valid. Maybe it could be make sense to use that here.

Reply:
@bisgaard-itis In reality the idea here is that provided some env vars in your system ... you are able to "extract" the minimum dotenv file that you need to run your service.   If some of the env vars are not present, we fill them with some "dummy" value